### DISCUSSION TITLE
Can we get FRIP after call peaks?

### DISCUSSION THREAD

**Discussion by Pengfei-Dong** [GreenleafLab/ArchR#1969] [Feature Requests]
Can we get FRIP after call peaks?
I have tried to calculate FRIP after getting the peak matrix, but found the colSums of the peak matrix are often large than the nFrags.

**Comment by rcorces** [comment]
Thats because a fragment has two reads

**Reply by Pengfei-Dong** [reply]
Thanks for the explanation!
